k8s/upgrades/0.8.0-0.8.1/jiva-replica-patch.tpl.json

```diff
@@ -0,0 +1,30 @@
+{
+   "metadata": {
+      "annotations": {
+         "openebs.io/storage-class-ref": "name: @sc_name\nresourceVersion: @sc_resource_version\n"
+      },
+      "labels": {
+         "openebs.io/version": "0.8.1"
+      }
+   },
+   "spec": {
+      "template": {
+         "metadata": {
+            "annotations": {
+               "openebs.io/storage-class-ref": "name: @sc_name\nresourceVersion: @sc_resource_version\n"
+            }
+         },
+         "spec": {
+            "containers": [
+               {
+                  "name": "@r_name",
+                  "image": "quay.io/openebs/jiva:0.8.1"
+               }
+            ],
+            "nodeSelector": {
```

Comment:
why do we want to force this?

Reply:
ok.. got to see the need of this..
this time, we should add this in CAS as well